### PR TITLE
ifdb-recommends.php: Delete legacy code

### PR DESCRIPTION
**Pull request by alice-blue**
Delete automatic recommendation algorithm that finds users who are “similar” to you and finds games that they recommend highly

### CODE REVIEW

File: www/components/ifdb-recommends.php [hunk 1]
```diff
@@ -4,293 +4,6 @@
...
-          }
-          xhr.send();
-        }();
-      </script>
-      <?php
-      return;
-    }
-}
 
 // if our recommendation list is empty, pick some random 4- and 5-star
```
Comment (dfabulich) [inline www/components/ifdb-recommends.php:8]: Let's fix this comment and remove the `if` statement

**Review by dfabulich — CHANGES_REQUESTED** [review]
Let's remove `$overloaded`, `$recsrc`, and `$_SESSION`, too.

File: www/components/ifdb-recommends.php (hunk 1 above)
Comment (alice-blue) [reply www/components/ifdb-recommends.php]: Ok. Does that look right?

File: www/components/ifdb-recommends.php [hunk 2]
```diff
@@ -3,300 +3,9 @@
...
-}
-
-// if our recommendation list is empty, pick some random 4- and 5-star
-// games as recommendations
-if (count($recs) == 0) {
-
-    if ($debugflag) echo "falling back on default method<br>";
+// Pick some random 4- and 5-star games as recommendations
 
     // if we're logged in, don't pick games we've already rated
```
Comment (dfabulich) [inline www/components/ifdb-recommends.php:10]: Thanks! Let's also remove the indentation that was here for the `if` block.

**Review by dfabulich — CHANGES_REQUESTED** [review]

File: www/components/ifdb-recommends.php (hunk 2 above)
Comment (alice-blue) [reply www/components/ifdb-recommends.php]: Ok, I think I got it.

**Pull request merged**